modules/ROOT/pages/client-id-based-policies.adoc

```diff
@@ -108,23 +133,19 @@ curl "http://localhost/myResource" -u 1234:abcd
 
 Example DataWeave 2.0 expression to be used when configuring the policy:
 
-No DataWeave 2.0 expression needed to be specified when the policy is configured to use the Basic Authentication as the origin of the credentials.
+To use the Basic Authentication as the origin of the credentials, no DataWeave 2.0 expression needs to be specified when configuring the policy.
 
-== WWW Authenticate HTTP header
-When a HTTP request is performed against a protected resource, where the Client ID enforcement policy is applied, and client application credentials are invalid or unauthorized, the HTTP response includes a WWW Authenticate header with the following values:
 
-[width="100%", cols="2,70"]
-|===
->s| Custom mode | Header 'WWW-Authenticate'='Client-ID-Enforcement'
->s| Basic Auth mode | Header 'WWW-Authenticate'='Basic realm="mule-realm"
-|===
+=== Configure API Specifications
+//What to add here? -- Eva (this is not needed?)
+
 
 == See Also
 
 * xref:rate-limiting-and-throttling-sla-based-policies.adoc[Rate Limiting - SLA-Based Policies concepts]
 * xref:policy-ping-federate.adoc[PingFederate OAuth 2.0 Token Enforcement]
 * xref:policy-openid-connect.adoc[OpenID Connect OAuth 2.0 Token Enforcement]
-* xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 Access Token]
+//* xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 Access Token]
 * xref:defining-sla-tiers.adoc[Reviewing SLA Tiers concepts on API Manager]
 * xref:manage-client-apps-latest-task.adoc[Managing API instance Contracts on API Manager]
 * xref:access-client-app-id-task.adoc[Getting Client ID and Client Secret of a registered Client App]
```

Comment:
I would order them differently, all the contract information first, since that is crucial for the workflow of this policy, and then you can add the links to the other policies that do internal enforcement of clients as well.

Reply:
Done! Thanks!